model/task/task.go

```diff
@@ -3364,6 +3356,15 @@ func getTasksByVersionPipeline(versionID string, opts GetTasksByVersionOptions)
 			},
 		})
 	}
+
+	if opts.IncludeBaseTasks && len(opts.BaseStatuses) > 0 {
```

Comment:
do we want to include this in the above` if opts.IncludeBaseTasks` ?  Do we specifically want it after the two if statements that follow?

Reply:
The base status filter depends on base status existing but base status does not depend on a filter existing.